bedrock/mozorg/templates/mozorg/contribute/index.html

```diff
@@ -108,13 +108,17 @@ <h3 class="fn given-name">Michael</h3>
 
   <section id="landing-howto" class="section landing-howto">
     <div class="container">
+      {# Hiding steps until we re-instate email signups
       <h2 class="section-title">{{ _('Want to get involved? Here’s how.') }}</h2>
       <ol class="steps">
         <li class="step1">{{ _('Choose what interests you and tell us how you’d like to help') }}</li>
         <li class="step2">{{ _('Stand by for an email that will connect you to the Mozilla community') }}</li>
         <li class="step3">{{ _('Create your Mozillians account and share your contributions with the world') }}</li>
       </ol>
       <p class="cta"><a href="{{ url('mozorg.contribute.signup') }}" class="button" data-link-type="button" data-link-name="Get started">{{ _('Get started') }}</a></p>
+      #}
+      <h2 class="section-title">{{ _('Join a global community of game-changers.') }}</h2>
```

Comment:
@alexgibson 
Just realized that this introduced a new string without a tag, since we have only `Join a global community of game-changersJoin a global community of game-changers` in the file (no final dot). Can we use the existing one? Otherwise we need a tag :-\